### PR TITLE
Option to set specific random seed for random number generator

### PR DESCRIPTION
**Pull request by elsid**
This one is more like debugging feature. It would be nice to have reproducible game situations. When random seed depends on time and engine makes a lot of decisions by rolling a dice this option can help. Probably it is not the best way to do it. I will listen other suggestions.

### CODE REVIEW

File: components/misc/rng.hpp [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 #include <cassert>
 #include <random>
+#include <chrono>
```
Comment (Capostrophic) [inline components/misc/rng.hpp]: A bunch of stuff depends on rng.hpp and including chrono in it is something scrawl advised against.

Comment (elsid) [reply components/misc/rng.hpp]: Ok, I will hide default seed generation into a function and will place definition into .cpp file.

**Comment by psi29a** [comment]
This still adds boost::chrono correct? Yet another external dependency... can we implement this without boost?

**Comment by psi29a** [comment]
Right, so this adds std::chrono and this is abstracted away... I'm OK with this, waiting for @Capostrophic opinion as well.

File: components/misc/rng.hpp (hunk 1 above)
Comment (psi29a) [reply components/misc/rng.hpp]: @Capostrophic is this OK?

File: apps/openmw/main.cpp [hunk 2]
```diff
@@ -3,9 +3,12 @@
 #include <components/files/escape.hpp>
 #include <components/fallback/validate.hpp>
 #include <components/debug/debugging.hpp>
+#include <components/misc/rng.hpp>
 
 #include "engine.hpp"
 
+#include <chrono>
```
Comment (Capostrophic) [inline apps/openmw/main.cpp]: This header doesn't seem to be necessary.

**Comment by Capostrophic** [comment]
Seems somewhat clunky but I'm not sure what could be better as a solution.

File: apps/openmw/main.cpp (hunk 2 above)
Comment (psi29a) [reply apps/openmw/main.cpp]: Agreed, this is only really needed for rng.cpp

Comment (elsid) [reply apps/openmw/main.cpp]: Fixed

**Comment by elsid** [comment]
Incude is fixed, also changelog record is added.

**Pull request merged**